src/NuGet.Services.AzureSearch/Db2AzureSearch/Db2AzureSearchCommand.cs

```diff
@@ -133,17 +140,29 @@ private async Task InitializeAsync()
             await _blobContainerBuilder.CreateAsync(containerDeleted);
             await _indexBuilder.CreateSearchIndexAsync();
             await _indexBuilder.CreateHijackIndexAsync();
+
+            try
+            {
+                var storageResult = await _auxiliaryFileClient.LoadExcludedPackagesListAsync(etag: null);
+                _excludedPackagesList = storageResult.Data ?? new HashSet<string>();
+            }
+            catch (StorageException ex) when(ex.RequestInformation?.HttpStatusCode == (int)HttpStatusCode.NotFound)
+            {
+                _logger.LogWarning($"Excluded packages list not found in the storage. No packages will be excluded.");
+                _excludedPackagesList = new HashSet<string>();
```

Comment:
>_excludedPackagesList [](start = 16, length = 21)

We should fail in this case

Reply:
I felt earlier we shouldn't.. but now I think that may be you are right, we should fail. Since if we were to miss adding exclusion list our prod results will be horrible.